ApplicationRequirements.txt

```diff
@@ -0,0 +1,139 @@
+	Displays "Delivery Instructions" Text
+	Displays "Pickup From" Text
+	Displays "Pickup Address" Text
+	Displays "Pickup Instructions" Text
+	Displays "What is Being Delivered" Text
+	Displays "Uploaded File" Text
+	"Confirm Order" Button
+		-> Takes user to "Confirmation" Screen
+		-> Adds request to the "Request Queue"
+		-> Sets the request status to "Awaiting Driver"
+		-> Adds request to the user's "Request History"
+	"Update Order" Button
+		-> Takes user to "Update Request" Screen
+		
+"Confirmation" Screen
+	Displays "Confirmation" Text
+	"Back to Home" Button
+		-> Takes user back to the "Home" Screen
+
+"Update Request" Screen
+	"Deliver To" Text Box
+		-> Has previous "Deliver To" information in it
+	"Delivery Address" Text Box
+		-> Has previous "Delivery Address" information in it
+	"Delivery Instructions" Text Box (Optional: For preferred delivery time and other special instructions)
+		-> Has previous "Delivery Instructions" information in it
+	"Pickup From" Text Box
+		-> Has previous "Pickup From" information in it
+	"Pickup Address" Text Box
+		-> Has previous "Pickup Address" information in it
+	"Pickup Instructions" Text Box (Optional: For preferred pickup time and other special instructions)
+		-> Has previous "Pickup Instructions" information in it
+	"Upload Files" Button (Optional: For uploading receipt/confirmation for online order)
+		-> Opens "File Explorer" to choose file
+		-> Has previous file in it
+	"Update" Button
+		-> Updates request information
+		-> Takes user to "Confirmation" Screen
+	"Cancel" Button
+		-> Retains request information
+		-> Takes user to "Checkout" Screen if they came from the "Checkout" Screen
+		-> Takes user to "My Requests" Screen if they came from the "My Requests" Screen
+		
+"Requests Available for Pickup" Screen
+	Shows the "Request Queue" of all submitted requests in a certain mile radius
+		-> Queue is ordered from oldest requests at the top to recently submitted at the bottom
+	Each request displays all of the information inputted when creating the request (deliver to, pickup from, etc.)
+	Each request has an "Accept Request" button
+		-> Clicking adds the request to the top of the driver's "My Deliveries" List
+		-> Clicking removes the request from the "Request Queue"
+		-> Clicking updates the request status to "Awaiting Pickup"
+		
+"My Deliveries" Screen
+	Shows the driver's "My Deliveries" List of all the deliveries that the driver has ever accepted
+	Each request shows the status of each request (en route, delivered, awaiting driver, canceled, etc.)
+	Each request shows all of the information inputted when creating the request (deliver to, pickup from, etc.)
+	Each request has an "Update Status" Dropdown that will change the status of the request
+		-> Driver is to select "En Route" after picking up the delivery
+		-> Driver is to select "Delivered" after completing the delivery
+		-> Driver can select "Release Request" before picking up the delivery
```

Comment:
What do you mean by release request?

Reply:
By "Release Request" , you mean the driver can essentially remove that specific delivery if need be ?

Reply:
Yes, so if something comes up and they can’t pick up the request they can release it for someone’s else’s to grab. Once they have an order in their possession they have to deliver it though.